datumaro/plugins/widerface_format.py

```diff
@@ -40,12 +41,14 @@ def __init__(self, path, subset=None):
 
     def _load_categories(self):
         label_cat = LabelCategories()
-
+        label_cat.add(WiderFacePath.DEFAULT_LABEL)
```

Comment:
Should this label be included if there is a file?

Reply:
You're right. This shouldn't work for a label file. Thanks.